wp-content/themes/pawar2018/page-issues.php

```diff
@@ -11,154 +11,41 @@
                 <p>President Roosevelt’s New Deal took our country out of depression and laid the foundation for decades of growth and prosperity. We can do the same thing in Illinois by rebuilding our crumbling infrastructure, reinvesting in good-paying jobs, and recommitting to early childhood education and public schools. And we can do it without further burdening the middle class.</p>
             </div>
-            <div class="feature-issue small-12 medium-10 large-5">
-                <img src="<?php echo get_bloginfo('template_url') ?>/assets/school.svg" class="issue-icon" alt="School Icon">
-                <div class="issue-summary">
-                    <h6>Education</h6>
-                    <p>Increase funding to all public schools by eliminating corporate tax loopholes and make millionaires pay their fair share.</p>
-                </div>
-            </div>
-            <div class="feature-issue small-12 medium-10 large-5">
-                <img src="<?php echo get_bloginfo('template_url') ?>/assets/family.svg" class="issue-icon" alt="Family Icon">
-                <div class="issue-summary">
-                    <h6>Childcare</h6>
-                    <p>Provide universal access to childcare and support working families with paid sick leave, fair scheduling practices, parental leave, and a living wage.</p>
-                </div>
-            </div>
-            <div class="feature-issue small-12 medium-10 large-5">
-                <img src="<?php echo get_bloginfo('template_url') ?>/assets/bridge.svg" class="issue-icon" alt="Bridge Icon">
-                <div class="issue-summary">
-                    <h6>jobs &amp; infrastructure</h6>
-                    <p>Create tens of thousands of new middle-class jobs with a New Deal infrastructure program.</p>
-                </div>
-            </div>
-            <div class="feature-issue small-12 medium-10 large-5">
-                <img src="<?php echo get_bloginfo('template_url') ?>/assets/hammer.svg" class="issue-icon" alt="Gavel Icon">
-                <div class="issue-summary">
-                    <h6>criminal justice reform</h6>
-                    <p>Pass criminal justice reform and refocus resources from prisons to diversion programs, job training and placement, and social and mental health services.</p>
+            <?php
+                $args = array(
+                'post_type' => 'pillars',
+                'orderby' => 'meta_value',
+                'order' => 'ASC') ?>
+              <?php $loop = new WP_Query( $args ); ?>
+              <?php while ( $loop->have_posts() ) : $loop->the_post(); ?>
+                <div class="feature-issue small-12 medium-10 large-5">
+                    <img src="<?php echo get_field('pillar_icon') ?>" class="issue-icon" alt="School Icon">
+                    <div class="issue-summary">
+                      <h6><?php the_title();?></h6>
+                      <p><?php the_content() ?></p>
+                    </div>
```

Comment:
Well it will no longer be used _after_ we switch the one page that uses it over to the new template. I am reticent to remove it when I'm not sure what happens to the page that it is assigned to before the switch, which is why it will stay until a future PR.